api-server/src/events/events.controller.ts

```diff
@@ -171,7 +168,7 @@ export class EventsController {
         verifiedOnly: true,
         startDate,
         endDate,
-        isUserAdmin,
+        request,
       })
```

Comment:
passing the request now as the logic is now a little more complicated than just isUserAdmin and it felt best to consolidate it in the service layer